rest-api/client_cli/rdr.py

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env python
+#
+# RDR cli tool launcher
+#
+
+# pylint: disable=superfluous-parens
+import copy
+import glob
+import importlib
+import os
+import sys
+
+lib_paths = ['../service_libs', 'service_libs']
+import_path = 'service_libs'
+
+
+def run():
+  args = copy.deepcopy(sys.argv)
+
+  show_usage = False
+  group = 'no-group'
+
+  # If help is select lets build a list of commands to show
+  if '--help' == sys.argv[1] or '-h' == sys.argv[1]:
+    show_usage = True
+
+  # If not showing help, get the group name and then we'll call it.
+  if not show_usage:
+    group = args.pop(1)
+    sys.argv = args
+
+  lp = None
+  for lib_path in lib_paths:
+    if os.path.exists(os.path.join(os.curdir, lib_path)):
+      lp = os.path.join(os.curdir, lib_path)
+
+  if not lp:
+    print('ERROR: service libs path not found, aborting.')
+    exit(1)
+
+  group_names = list()
+
+  libs = glob.glob(os.path.join(lp, '*.py'))
+  for lib in libs:
+
+    mod_name = os.path.basename(lib).split('.')[0]
+    mod = importlib.import_module('{0}.{1}'.format(import_path, mod_name))
+
+    if hasattr(mod, 'group'):
+
+      if show_usage:
+        if mod.group != 'template':
+          group_names.append('  {0} : {1}'.format(mod.group.ljust(14), mod.group_desc))
+      else:
+        if mod.group == group:
```

Comment:
I'm not hung up on using 'group', in fact I don't think its a great name at all.  I was following the Google cli tool convention.  If you run 'gcloud --help' or 'gsutil --help', you'll see they use 'group' as the option name.  If our users are familiar with any of the google cli tools, then our cli tool convention should be easy for them to understand.

Reply:
I'd approve this with `group` as a name if we can allow it to run multiple items in that _group_ (anything with the same name) otherwise I'd like to change this to `command` (or something to that effect). Either one of those would resolve the ambiguity.

Reply:
If we remove the break at line 57 it will run any file that matches the group name, instead of just the first.  I'm good either way, do you want to change the name from 'group' to 'command' or run multiple files?

Reply:
If we remove the break at link 57 it will run all files with that match the group name, instead of just the first one.  I'm good either way, which would you prefer?

Reply:
I think it would be safer for us to use `command` and only allow one (just to prevent unintentional scripts from running because of a copy-paste error)